component-library/src/app/pages/banner-documentation/banner-documentation.component.scss

```diff
@@ -0,0 +1,267 @@
+@use '../../../../../core-library/util/device' as device;
+
+.content-doc-wrapper {
+  box-sizing: border-box;
+  max-width: 900px;
+  margin: 0 auto;
+
+  img {
+    width: 100%;
+  }
+}
+
+.body1.intro {
+  margin-top: 0px;
+  margin-bottom: 0px;
+}
+
+.body3.demo {
+  margin-top: 0px;
+  margin-bottom: 24px;
+}
+
+.sub-headings {
+  margin-top: 0px;
+  margin-bottom: 12px;
```

Comment:
This need to update: `margin-bottom: 36px;`

Reply:
Fixed